### DISCUSSION TITLE
Build failure output twice

### DISCUSSION THREAD

**Discussion by jonnygrant** [compiler-explorer/compiler-explorer#4917] [General]
Hello everyone!
Not sure if this is something I am doing wrong. When I enable "execute the code" from the tools cog wheel, I get the compile failure output twice.

This is just an example, I do expect it to fail, and hoped it would only show once like when building on the command line.

https://godbolt.org/z/dvhxed7zv

The build failed, so it didn't manage to execute the code.

**Comment by partouf** [comment]
This is expected behavior. Building an executable is a separate process and independent of the compilation in order to show assembly - and so you sometimes get the same error twice. Nothing you did wrong.

**Reply by jonnygrant** [reply]
ah ok, I was expecting when Execute was enabled, it would compile to asm, and then assemble and link. But even if it is a separate process, it could just not show any of the first messages from when the process that makes the asm is run?
